app/models/sam_account_reckoner.rb

```diff
@@ -0,0 +1,20 @@
+class SamAccountReckoner < Struct.new(:user)
```

Comment:
Do you think we should namespace these within some module like we do for the Presenters? Maybe something like `Decorator::SamAccountReckoner` or is that not really that big a deal?